osu.Desktop/Program.cs

```diff
@@ -25,17 +26,22 @@ public static class Program
         public static int Main(string[] args)
         {
             LegacyFilesystemReader.Register();
+            
+            var cwd = Directory.GetCurrentDirectory();

```

Comment:
`Environment.CurrentDirectory` may be a better fit? not sure what you're using this for though. needs comments since it's not obvious.